web/client/components/data/query/AutocompleteFieldHOC.jsx

```diff
@@ -56,6 +56,10 @@ class AutocompleteFieldHOC extends React.Component {
     };
 
     getPagination = (options) => {
+        if (!this.props.filterField.options) {
```

Comment:
hi, Can you add a unit test?
in this case seems needed to prevent future  run time error

Reply:
@MV88 test is provided